fzf-tab.zsh

```diff
@@ -184,6 +188,7 @@ _fzf_tab_get_headers() {
 
 # pupulates array `candidates` with completion candidates
 _fzf_tab_get_candidates() {
+    setopt localoptions extendedglob
```

Comment:
Instead of this, replace `emulate -L zsh` in `_fzf_tab_complete` with `emulate -L zsh -o extended_glob`.